examples/sdk/aqa-typescript-auth-example/README.md

```diff
@@ -0,0 +1,46 @@
+# Basic TypeScript and NPM example of SDK usage
+
+This basic example shows how simple it is to make a TypeScript integration, as well as using the SDK via NPM instead of a script.
```

Comment:
As I see you have integration of SDK via script tag instead of npm. Should we adjust description?